style.css

```diff
@@ -0,0 +1,9 @@
+.title {
+  font-family: PT Serif, Arial, sans-serif;;
+  font-weight: 700;
+}
+
+.subtitle {
+  font-family: Roboto, Arial, sans-serif;;
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6280509)